CHANGELOG.md

```diff
@@ -1,4 +1,81 @@
+* compute: added `replica_names` field to `sql_database_instance` resource ([#8637](https://github.com/hashicorp/terraform-provider-google-beta/pull/8637))
+* compute: added new field `instance_flexibility_policy` to resource `google_compute_region_instance_group_manager` ([#8581](https://github.com/hashicorp/terraform-provider-google-beta/pull/8581))
+* compute: increased `google_compute_security_policy` timeouts from 20 minutes to 30 minutes ([#8589](https://github.com/hashicorp/terraform-provider-google-beta/pull/8589))
+* container: added `control_plane_endpoints_config` field to `google_container_cluster` resource. ([#8630](https://github.com/hashicorp/terraform-provider-google-beta/pull/8630))
+* container: added `parallelstore_csi_driver_config` field to `google_container_cluster` resource. ([#8607](https://github.com/hashicorp/terraform-provider-google-beta/pull/8607))
+* container: added `user_managed_keys_config` field to `google_container_cluster` resource. ([#8562](https://github.com/hashicorp/terraform-provider-google-beta/pull/8562))
+* firestore: allowed single field indexes to support `__name__ DESC` indexes in `google_firestore_index` resources ([#8576](https://github.com/hashicorp/terraform-provider-google-beta/pull/8576))
+* privateca: added support for sub-CA to be activated into STAGED state ([#8560](https://github.com/hashicorp/terraform-provider-google-beta/pull/8560))
+* spanner: added `default_backup_schedule_type` field to  `google_spanner_instance` ([#8644](https://github.com/hashicorp/terraform-provider-google-beta/pull/8644))
+* vertexai: added `traffic_split`, `private_service_connect_config`, `predict_request_response_logging_config`, `dedicated_endpoint_enabled`, and `dedicated_endpoint_dns` fields to `google_vertex_ai_endpoint` resource ([#8619](https://github.com/hashicorp/terraform-provider-google-beta/pull/8619))
+* workflows: added `deletion_protection` field to `google_workflows_workflow` resource ([#8563](https://github.com/hashicorp/terraform-provider-google-beta/pull/8563))
+
+BUG FIXES:
+* compute: fixed a diff based on server-side reordering of `match.src_address_groups` and `match.dest_address_groups` in `google_compute_network_firewall_policy_rule` ([#8592](https://github.com/hashicorp/terraform-provider-google-beta/pull/8592))
+* compute: fixed permadiff on the `preconfigured_waf_config` field for `google_compute_security_policy` resource ([#8622](https://github.com/hashicorp/terraform-provider-google-beta/pull/8622))
+* container: fixed in-place updates for `node_config.containerd_config` in `google_container_cluster` and `google_container_node_pool` ([#8566](https://github.com/hashicorp/terraform-provider-google-beta/pull/8566))
+* looker: made `oauth_config` a required field in `google_looker_instance`, as creating this resource without that field always triggers an API error ([#8633](https://github.com/hashicorp/terraform-provider-google-beta/pull/8633))
```

Comment:
Should this be a bug fix or marked as a larger change as it's technically breaking?

Reply:
Looks like we've done it both ways in the past. Seems fine to call it out more strongly.